src/license/license.proto

```diff
@@ -43,7 +43,14 @@ message AddClusterRequest {
   // to authenticate to the license server. Otherwise a secret will be
   // generated and returned in the response. 
   string secret = 3;
+  // The pachd address for users to connect to. 
+  string user_address = 4;
+  // The deployment ID value generated by each Cluster
+  string cluster_deployment_id = 5;
+  // the enterprise server flags whether this cluster is registered with the enterprise server
```

Comment:
The comment is kind of misleading, this field indicates whether the address points to an enterprise server.